### DISCUSSION TITLE
formily-editor示例报错

### DISCUSSION THREAD

**Discussion by Z-Champion** [alibaba/formily#1312] [Q&A 2.x] [answered]
sandbox报错
![image](https://user-images.githubusercontent.com/33289026/116215706-812d2400-a77a-11eb-9695-a35085cc4b7f.png)

**Comment by Z-Champion** [comment]
https://codesandbox.io/s/formlily-schema-editor-vb0ch

**Reply by Z-Champion** [reply]
编辑器报错的

**Comment by janryWang** [comment, marked as answer]
https://github.com/alibaba/formily/discussions/1283

**Reply by Z-Champion** [reply]
thanks

**Reply by Z-Champion** [reply]
能不能支持一下4.0
